config.toml

```
@@ -352,6 +352,10 @@ logo = "schreder-hyperion.png"
 url = "https://www.teamleader.eu"
 logo = "teamleader.png"
 
+[[params.users]]
+url = "https://www.theproindia.com"
+logo = "assets/images/pro-logo.svg"
```

The image needs to be added to https://github.com/kedacore/keda-docs/tree/master/static/img/logos

```suggestion
logo = "pro-logo.svg"
```